python/setup.cfg

```diff
@@ -28,26 +28,4 @@ install_requires = cffi>=1.0.0
 opentracing = opentracing; jaeger_client
 
 [flake8]
-# TODO(PROTON-2322) decrease the limit
-max-line-length = 150
-
-# TODO(PROTON-2322) re-enable all these warnings
-ignore =
-    # do not use bare 'except'
-    E722,
-    # imported but unused
-    F401,
-    # ambiguous variable name 'l'
-    E741,
```

Comment:
don't like this warning, would've kept it disabled

Reply:
I agree that I wouldn't put it in a coding standard myself - but forbidding the letter 'l' to avoid confusion eith the number '1' has some sense IMO and it is in PEP8.

Again it's easy enough to avoid if you're checking before commit.